spec/requests/complaints_spec.rb

```diff
@@ -51,7 +51,7 @@
 
           get complaints_path
 
-          expect(response.body).to include '<table class="usa-table" aria-describedby="#caption" >'
+          expect(response.body).to include '<table class="usa-table">'
```

Comment:
[question] i think's good to have some sort of screen-readable description of the table. what was the impetus behind removing this?

Reply:
the caption was moved within the table (to fix a JS error that was occurring on sort around the caption not being found), so (I believe) the screen-readable description can be found without an `aria-` attribute.